### ISSUE TITLE
Enable automatic OSM sync for a standard set of Layer

### ISSUE THREAD

**Issue by vdeparday** [ebola-geonode/ebola_geonode#35]
Enable an hourly automatic hourly refresh of OSM data for basic layers 
@state-hiu has stock mappings that can be used already

**Comment by jj0hns0n** [comment]
@state-hiu @ScottEvil how much of this matches the OCHA styles?

**Comment by ScottEvil** [comment]
I don't think it would have any overlap. The OCHA symbols are different topic areas.

**Comment by state-hiu** [comment]
I've converted some of the OSM Humanitariian Basemap to SLD in our styles repo at https://github.com/state-hiu/cybergis-styles.  However, I need significantly refractor the repo to add in another layer of abstraction (read: one more layer in the folder structure).
